common/src/main/java/com/datastrato/gravitino/dto/requests/SchemaUpdateRequest.java

```diff
@@ -30,6 +33,35 @@ public interface SchemaUpdateRequest extends RESTRequest {
 
   SchemaChange schemaChange();
 
+  @EqualsAndHashCode
+  @ToString
+  class UpdateSchemaCommentRequest implements SchemaUpdateRequest {
+
+    @Getter
+    @JsonProperty("newComment")
+    private final String newComment;
+
+    public UpdateSchemaCommentRequest(String newComment) {
+      this.newComment = newComment;
+    }
+
+    public UpdateSchemaCommentRequest() {
+      this(null);
+    }
+
+    @Override
+    public void validate() throws IllegalArgumentException {
+      Preconditions.checkArgument(
+          StringUtils.isNotBlank(newComment),
```

Comment:
Is it sufficient to check for null here, or is an empty string  `" "` also not allowed?

Reply:
@mchades, the `UpdateTableColumnCommentRequest` validation behavior also validates whether `newComment` is blank.

Reply:
I mean we should allow users setting schema comment to `" "`